lib/hades_web/router.ex

```diff
@@ -22,6 +22,7 @@ defmodule HadesWeb.Router do
     pipe_through [:api, :authorized]
 
     get "/users/:id", UserController, :get_user
+    put "/user/update_status", UserController, :update_user_status
```

Comment:
I think it's weird to have an action name in the endpoint, like `update`, since the HTTP verb express that, WDYT? Maybe we could have just `put "/user/status"`

Reply:
Makes sense! Thanks for the tip!